### DISCUSSION TITLE
Will you attend physically the ICRA 2023 finals in London?

### DISCUSSION THREAD

**Discussion by omichel** [cyberbotics/wrestling#78] [General]
If any participant would like to attend physically the finals during the ICRA 2023 conference in London, please let me know here, so that I can officially register your name. Of course, physical attendance is not mandatory.

**Comment by christopherpeers** [comment]
ill be there

**Reply by omichel** [reply]
Great! I am looking forward to meet you in person.
Could you please send me your full name and e-mail address at Olivier.Michel@cyberbotics.com? I need it to register you officially at ICRA 2023.

**Comment by AndrejOrsula** [comment]
Hello, I would like to attend. Unfortunately, I am able to come only on the first day (May 30th).
I sent an email with my details to you @omichel. Thank you.

**Reply by omichel** [reply]
No problem, you are welcome. I just replied to your e-mail.
